contracts/dataAccessV1/accessSettlement/interfaces/IAccessSettlement.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+/**
+ * @title IAccessSettlement
+ * @notice Interface for the AccessSettlement contract
+ * @dev Acts as a trustless clearinghouse for data access and compute operations
+ */
+interface IAccessSettlement {
+    /**
+     * @notice State of a completed but unpaid operation
+     * @param issuer The address that gets paid
+     * @param grantee The address that needs to pay
+     * @param price Price for the operation
+     * @param tokenAddress Token address for payment (address(0) for native VANA)
+     * @param isSettled Whether payment has been settled
+     */
+    struct OperationInvoice {
+        address issuer;
+        address grantee;
+        uint256 price;
+        address tokenAddress;
+        bool isSettled;
+    }
+
+    /**
+     * @notice Emitted when an operation is ready for settlement
+     * @param operationId Unique identifier for the operation
+     * @param issuer Address that will receive payment
+     * @param grantee Address that needs to pay
+     * @param price Price for the operation
+     * @param tokenAddress Token address for payment
+     */
+    event OperationReadyForSettlement(
+        bytes indexed operationId,
```

Comment:
What's the intention of typing this as `bytes` rather than `string`?

What is the guidance for how operationIds should be constructed?